Baconit/Strings/es-es/Resources.resw

```diff
@@ -0,0 +1,889 @@
+  </data>
+  <data name="MSBand3.Text" xml:space="preserve">
+    <value>Fondo de pantalla</value>
+  </data>
+  <data name="MSBand4.Text" xml:space="preserve">
+    <value>Automáticamente actualiza el fondo de pantalla de Band</value>
+  </data>
+  <data name="MSBand5.Text" xml:space="preserve">
+    <value>Origen del fondo de pantalla de Band</value>
+  </data>
+  <data name="MSBand6.Text" xml:space="preserve">
+    <value>Azulejo &amp; Entrada</value>
+  </data>
+  <data name="MSBand7.Text" xml:space="preserve">
+    <value>Habilita la actualización en segundo plano de la bandeja de entrada para usar esta característica.</value>
+  </data>
+  <data name="NeverCombo.Content" xml:space="preserve">
+    <value>Nunca</value>
+  </data>
+  <data name="NNewCB.Content" xml:space="preserve">
+    <value>Nueva</value>
+  </data>
+  <data name="NSFWB.Text" xml:space="preserve">
+    <value>Filtro de contenido para adultos</value>
+  </data>
+  <data name="PinToStart.Content" xml:space="preserve">
+    <value>Enlaza al comenienzo.</value>
+  </data>
+  <data name="PostsCB.Content" xml:space="preserve">
+    <value>Posts</value>
+  </data>
+  <data name="PreloadComments.Text" xml:space="preserve">
+    <value>Precarga los comentarios</value>
+  </data>
+  <data name="PreloadPost.Text" xml:space="preserve">
+    <value>Precarga contenido de los futuros post.</value>
+  </data>
+  <data name="RateUnder.Text" xml:space="preserve">
+    <value>Valora y comenta en la Windows Store</value>
+  </data>
+  <data name="RelevanceCB.Content" xml:space="preserve">
+    <value>Relevancia</value>
+  </data>
+  <data name="ResetReview.Content" xml:space="preserve">
+    <value>Reinicia el contador de valoraciones y comentarios.</value>
+  </data>
+  <data name="SearchThisSubReddit" xml:space="preserve">
+    <value>Buscar esto en subreddit.</value>
+  </data>
+  <data name="SelfFilter.PlaceholderText" xml:space="preserve">
+    <value>Filtro de autotexto.</value>
+  </data>
+  <data name="SelfPostCB.Content" xml:space="preserve">
+    <value>¿Es un post propio?</value>
+  </data>
+  <data name="SelfPostNCB.Content" xml:space="preserve">
+    <value>No</value>
+  </data>
+  <data name="SelfPostYCB.Content" xml:space="preserve">
+    <value>Sí</value>
+  </data>
+  <data name="SendCT.Content" xml:space="preserve">
+    <value>Enviar</value>
+  </data>
+  <data name="ShowUITB.Text" xml:space="preserve">
+    <value>Mostrar la ayuda de la interfaz de usuario.</value>
+  </data>
+  <data name="SubFilter.PlaceholderText" xml:space="preserve">
+    <value>Filtro de Subreddit</value>
+  </data>
+  <data name="SubmitNewPost.Content" xml:space="preserve">
+    <value>Enviar un nuevo post.</value>
+  </data>
+  <data name="SubredditsCB.Content" xml:space="preserve">
+    <value>Subreddits</value>
+  </data>
+  <data name="SubscribeCT.Content" xml:space="preserve">
+    <value>Subscribirse</value>
```

Comment:
Even that "subscribirse" is a valid word, "suscribirse" (without the B) has being more recently used.